specification/service-search-api.yaml

```diff
@@ -291,15 +291,22 @@ paths:
             type: string
       responses:
         "200":
+          description: Successful response
           $ref: "#/components/responses/SearchResponse"
-        "400":
-          $ref: "#/components/responses/BadRequest"
-        "401":
-          $ref: "#/components/responses/Unauthorized"
-        "404":
-          $ref: "#/components/responses/NotFound"
-        "429":
-          $ref: "#/components/responses/TooManyRequests"
+        "4XX":
+          $ref: "#/components/responses/4XX-SERVICE-SEARCH"
+#        "400":
```

Comment:
Is there a reason these are commented? If not needed please remove them.

Reply:
done